Ported math plugin from dev to es6

avipatel91 opened the pull request:
Math plugin evaluates expressions using mathjs. If an invalid expression
is provided, 'invalid input' is returned.

Commit:
Ported math plugin from dev to es6

Math plugin evaluates expressions using mathjs. If an invalid expression
is provided, 'invalid input' is returned.

avipatel91 commented:
It is failing because there is no regex match for messages (e.g. ping) without backslashes in Plugin manager.
[regex location](https://github.com/crisbal/Telegram-Bot-Node/blob/es6/src/PluginManager.js#L184l)

CapacitorSet commented:
Yes, don't worry about the CI build, if you look at the build history you'll see that it has been failing for some time. I tried fixing that in 5be52071ecdf2ef9b415f2c015d7abe9e1e9aa7e, but it didn't work; if you want to help with that, PRs are certainly welcome.

CapacitorSet commented on src/plugins/Math.js:
```diff
@@ -0,0 +1,33 @@
+            type: "text",
+            text: result,
+            options: {
+                parse_mode: "markdown"
```
Why is this line needed?

avipatel91 replied:
To be honest, I do not know. I saw it in the original and copied it over. But now that I look at it, the original had a template literal. Markdown shouldn't really be required for this since I am using a string without any markdowns. I can remove it.

CapacitorSet replied:
Please remove it, and then the PR will be ready for merging into `es6`.

Commit:
Removed unecessary options.

Removed markdown option for math plugin because text is not using any
markdown.

CapacitorSet commented:
Thanks!
